tilings/tiling.py

```diff
@@ -902,7 +904,8 @@ def is_empty(self) -> bool:
             return True
```

Comment:
I think those two edge cases are now covered by MinimalGriddedPerms

Reply:
I agree, but it prevents creating an MGP instance.

Reply:
Good, not very important anyway